utils/parse_test.go

```diff
@@ -23,13 +21,28 @@ func TestParseShell(t *testing.T) {
 		"'one two three'":   []string{"one two three"},
 		"‘one two three’":   []string{"one two three"},
 		"“one two three”":   []string{"one two three"},
-		"one 'two three":    []string{"one", "'two", "three"},
-		"one \"two three":   []string{"one", "\"two", "three"},
+		"one 'two three":    []string{"one 'two three"},
 	}
 
 	for input, expected := range cases {
 		t.Run(input, func(t *testing.T) {
-			assert.Equal(t, expected, ParseShell(input))
+			if out, err := ParseShell(input); err != nil {
```

Comment:
this is not the behavior we want to test; please put it back to the way it was before

Reply:
I'm not entirely sure how to otherwise do this given the multi-return of `ParseShell`. `expected` is the value of the map keyed on `input` and we're asserting that `expected is equal` to the return of `ParseShell` which is a mismatch.

Thus, this is why I split `out` and `err` before asserting on line 30.

Reply:
```
result, err := doThing(...)
if err != nil {
    t.Fatal(err)
}

assert.Equal(t, expected, result)
```